lib/platform/github/index.ts

```diff
@@ -619,11 +617,9 @@ async function getOpenPrs(): Promise<PrList> {
         }
       }
       `;
-      const nodes = await githubApi.getGraphqlNodes<any>(
-        query,
-        'pullRequests',
-        { paginate: false }
-      );
+      const nodes = await githubApi.queryRepoField<any>(query, 'pullRequests', {
+        paginate: false,
```

Comment:
FYI I'm surprised to see pagination false for these

Reply:
I'll enable it in separate PR then

Reply:
I'm trying to think why it was left with no pagination though